src/entities/IUse.ts

```diff
@@ -61,4 +61,8 @@ export enum Activities {
   FishingVessel = "FISHING_VESSEL",
   ClimbingMountaineering = "CLIMBING_MOUNTAINEERING",
   Glider = "GLIDER",
+  SailingVessel = "SAILING_VESSEL",
+  Other = "OTHER",
+  WorkingRemotely = "WORKING_REMOTELY",
+  Windfarm = "WINDFARM",
```

Comment:
I have only added in the additional activities required for the `Land` view and those used in tests